redfish/volume.go

```diff
@@ -192,6 +192,15 @@ type Volume struct {
 	drives []string
 }
 
+// Volumes is used to represent the volumes information related to a Storage
```

Comment:
The storage.volumes property is actually a VolumesCollection. That collection is access via the existing GetReferencedVolumes call below.

The existing volume struct should be used rather creating a duplicate here. See bios.go for an existing example of extracting ApplyTimes so we can stay consistent with how this is done.

Reply:
Hi @stmcginnis ,

I think it is a little bit different from the bios case. In the Bios case we have an struct where we unmarshal the response from the redfish API. Calling to */redfish/v1/Systems/System.Embedded.1/Bios* gives you, among other things, the *@Redfish.Settings* for that.

In the case of Volumes, the endpoint is */redfish/v1/Systems/System.Embedded.1/Storage/RAID.Integrated.1-1/Volumes* and produces a result like this one:
~~~
{
	"@Redfish.OperationApplyTimeSupport": {
		"@odata.type": "#Settings.v1_2_1.OperationApplyTimeSupport",
		"SupportedValues": [
			"Immediate",
			"OnReset"
		]
	},
	"@odata.context": "/redfish/v1/$metadata#VolumeCollection.VolumeCollection",
	"@odata.id": "/redfish/v1/Systems/System.Embedded.1/Storage/RAID.Integrated.1-1/Volumes",
	"@odata.type": "#VolumeCollection.VolumeCollection",
	"Description": "Collection Of Volume",
	"Members": [
		{
			"@odata.id": "/redfish/v1/Systems/System.Embedded.1/Storage/RAID.Integrated.1-1/Volumes/Disk.Virtual.0:RAID.Integrated.1-1"
		}
	],
	"Members@odata.count": 1,
	"Name": "Volume Collection"
}
~~~

It gives you the *Redfish.OperationApplyTimeSupport*. That value (to my understanding) gives you the supported operations on that disk controller. I need those values.
Take into account that I'm verifying that on the Volumes endpoint before creating a volume (Volumes collection is empty at that point).

Using the current Volume structure would be like querying */redfish/v1/Systems/System.Embedded.1/Storage/RAID.Integrated.1-1/Volumes/Disk.Virtual.0:RAID.Integrated.1-1*, but I don't want to do that.

How could I possibly get *@Redfish.OperationApplyTimeSupport* on */redfish/v1/Systems/System.Embedded.1/Storage/WHATEVER_CONTROLLER/Volumes* endpoint as per the current implementation?

Thanks!
/Miguel